### PR TITLE
Address a couple of low hanging warnings

### PR DESCRIPTION
**Pull request by mokagio**
Noticed these while looking through the codebase. They were straightforward to fix.

## Testing

CI will suffice.

## Regression Notes

1. Potential unintended areas of impact – N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on) – N.A.
3. What automated tests I added (or what prevented me from doing so) – N.A.

---

**PR submission checklist:**

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

**UI changes testing checklist:** Not a UI PR.

### CODE REVIEW

File: WordPress/Classes/Services/MediaService.h [hunk 1]
```diff
@@ -59,7 +59,7 @@ typedef NS_ERROR_ENUM(MediaServiceErrorDomain, MediaServiceError) {
  @failure a block that will be invoked when there is upload error.
  */
 - (void)updateMedia:(nonnull Media *)media
-     fieldsToUpdate:(NSArray<NSString *> *)fieldsToUpdate
+     fieldsToUpdate:(nonnull NSArray<NSString *> *)fieldsToUpdate
```
Comment (mokagio) [inline WordPress/Classes/Services/MediaService.h:62]: I made the assumption that one would never want to call this method by passing `nil` to this parameter.  An the compiler didn't reveal any inconsistency.

I was happy to see the method also had _some_ test coverage

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift [hunk 2]
```diff
@@ -293,8 +293,24 @@ class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelega
...
+            loadAutosaveRevision: loadAutosaveRevision,
+            replaceEditor: replaceEditor,
+            editorSession: editorSession,
+            // Notice this parameter.
+            // The value is the default set in the required init but we need to set it explicitly,
+            // otherwise we'd trigger and infinite loop on this init.
+            //
+            // The reason we need this init at all even though the other one does the same job is
+            // to conform to the PostEditor protocol.
+            navigationBarManager: nil
```
Comment (mokagio) [inline WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift:312]: This is the fix, the rest of the diff in the file is just splitting long lines.

**Review by hassaanelgarem — APPROVED** [review]
LGTM 🚀

**Review by mkevins — COMMENTED** [review]
I also briefly tested this PR in the simulator, by opening the Gutenberg editor, and things are working as expected. Thanks for the fixes!

**Pull request merged**